app/models/dialog.rb

```diff
@@ -153,6 +160,12 @@ def deep_copy(new_attributes = {})
 
   private
 
+  def create_association_list(associations_on_dialog_fields, dialog_field_list)
+    associations_on_dialog_fields.each_with_object({}) do |a, associations|
+      associations[dialog_field_list.find { |df| df.id == a.trigger_id }.name] = dialog_field_list.select { |df| df.id == a.respond_id }.pluck(:name)
```

Comment:
This can't be used in a validation since it's dependent on `DialogField`s already having `id`s (which implies already being saved)

Reply:
They are.

Reply:
I would hope they're not yet saved on a brand new dialog (which is not yet saved).

Reply:
Is this being done in a large transaction or something?